signer/tests/integration/transaction_coordinator.rs

```diff
@@ -4709,3 +4711,59 @@ mod get_eligible_pending_withdrawal_requests {
         testing::storage::drop_db(db).await;
     }
 }
+
+#[test_log::test(tokio::test)]
+async fn should_handle_dkg_coordination_failure() {
```

Comment:
I don't really see why we will experience dkg failure in this test. Maybe add some comments about it

Reply:
Ohhh, you rebased, never mind